vindbaarheid/vindbaarheid.shacl.ttl

```diff
@@ -0,0 +1,492 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix schema: <https://schema.org/> .
+@prefix premis: <http://www.loc.gov/premis/rdf/v3/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix meemoo: <http://data.meemoo.be/ns/>  .
+@prefix ebu: <http://www.ebu.ch/metadata/ontologies/ebucore/ebucore#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix org: <http://www.w3.org/ns/org#> .
+
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+##############################
+#         NodeShapes         #
+##############################
+
+####################
+#      PREMIS      #
+####################
+premis:IntellectualEntity a rdfs:Class, sh:NodeShape ;
```

Comment:
ik weet dat ik dit in de bestaande documenten heb gedaan, maar het is beter, zeker omdat je nu een aparte RDFS hebt, om `<#IntellectualEntityNodeShape> a sh:NodeShape ; sh:targetClass premis:IntellectualEntity` te schrijven. Zo zijn uw RDFS en SHACL onafhankelijk.